### PR TITLE
Add Spanish version and support Deno tasks

### PR DESCRIPTION
**Pull request by jheysaaz**

### CODE REVIEW

File: server/template.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { urlParamsNames } from "./params.ts";
 import { flagsNormalization } from "./dictionary.ts";
 
 const style = await Deno.readTextFile(
-  new URL("../static/main.css", import.meta.url)
+  new URL("../static/main.css", import.meta.url),
```
Comment (nicolo-ribaudo) [inline server/template.ts]: Are these changes generated by `deno fmt`? (I'm _very_ new to Deno: I know that it has a formatter, but I never used it)

File: server/template.ts [hunk 2]
```diff
@@ -68,11 +68,9 @@ function buildHead({ lang }: Data) {
       <link rel="icon" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 100 100%22><text y=%22.9em%22 font-size=%2290%22>💌</text></svg>">
 
       ${"" /* <link rel="stylesheet" href="./main.css" /> */}
-      ${
-        ""
-        /* We inline styles to prevent transitions from running on page load when the
+      ${""}/* We inline styles to prevent transitions from running on page load when the
         page is rendered before loading the stylesheets. */
-      }
```
Comment (nicolo-ribaudo) [inline server/template.ts]: This is wrong, it moved the comment inside the template contents :thinking:

Comment (nicolo-ribaudo) [reply server/template.ts]: I'll report this to the correct repository.

**Comment by nicolo-ribaudo** [comment]
Could you update this PR to _not_ reformat the code? :pray: I'll take care of that when the formatting bug is fixed.

File: server/template.ts (hunk 2 above)
Comment (nicolo-ribaudo) [reply server/template.ts]: https://github.com/dprint/dprint-plugin-typescript/pull/393

File: server/template.ts (hunk 1 above)
Comment (jheysaaz) [reply server/template.ts]: Yes, this changes are generated by `deno fmt`

**Comment by jheysaaz** [comment]
> Could you update this PR to _not_ reformat the code? pray I'll take care of that when the formatting bug is fixed.

I'm on it :+1:

**Comment by jheysaaz** [comment]
Done, I removed the formatting and added Spanish version and the `serve` Deno task

**Comment by nicolo-ribaudo** [comment]
Thank you!

**Pull request merged**